Adds event-handler systemd example (#16460)
gravitational/teleport: docs/pages/management/guides/fluentd.mdx

...
```

> ## Setup with systemd

Reviewer: This this section be numbered, and the sections above have their `/6` updated?

> The above steps are an example Fluentd configuration. In production, we 
> recommend starting the Teleport plugin daemon via an init
> system like systemd. First copy `teleport-event-handler` to `/usr/local/bin`. Second
> update the `teleport-event-handler.toml` file to use fully qualified paths and copy into `/etc/`.

Reviewer: ```suggestion
The above steps are an example Fluentd configuration. In production, we 
recommend starting the Teleport plugin daemon via an init system like
 systemd.
 
First, copy `teleport-event-handler` to `/usr/local/bin`. Then update the
`teleport-event-handler.toml` file to use fully qualified paths, and move
it into `/etc/`.
```

> Here's the recommended Teleport plugin service unit file
> for systemd:

Reviewer: Hadn't hit 80 chars:

```suggestion
Here's the recommended Teleport plugin service unit file for systemd:
```

```ini

> (!examples/systemd/plugins/teleport-event-handler.service!)

Reviewer: I assume this is an include because we have or expect to have other pieces of content where this example can be reused?

```

> Save this as `teleport-event-handler.service` in either `/usr/lib/systemd/system/` or
> another [unit file load
> path](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Unit%20File%20Load%20Path)
> supported by systemd.

Reviewer: ```suggestion
Save this as `teleport-event-handler.service` in either `/usr/lib/systemd/system/`
or another [unit file load path](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Unit%20File%20Load%20Path)
supported by systemd.
```

Enable and start the plugin:

```code

> $ sudo systemctl enable teleport-event-handler
> $ sudo systemctl start teleport-event-handler

Reviewer: ```suggestion
$ sudo systemctl enable --now teleport-event-handler
```
